src/notifications/builder.ts

```diff
@@ -213,6 +228,12 @@ export class NotificationBuilder {
     }).avatarManyUsers(upvoters);
   }
 
+  numTotalAvatars(numTotalAvatars: number): NotificationBuilder {
+    return this.enrichNotification({
```

Comment:
Don't we want to set it in `avatarManySources`?

Reply:
Could probably put it in there, but I split them so that they can be re-used